src/omv/ports/stm32/sensor.c

```diff
@@ -218,13 +218,17 @@ static void dcmi_abort()
     if (DCMI->CR & DCMI_CR_ENABLE) {
         DCMI->CR &= ~DCMI_CR_ENABLE;
         HAL_DMA_Abort(&DMAHandle);
-        __HAL_DCMI_DISABLE_IT(&DCMIHandle, DCMI_IT_FRAME);
+        HAL_NVIC_EnableIRQ(DMA2_Stream1_IRQn);
         #if (OMV_ENABLE_SENSOR_MDMA == 1)
         HAL_MDMA_Abort(&DCMI_MDMA_Handle0);
         HAL_MDMA_Abort(&DCMI_MDMA_Handle1);
         HAL_MDMA_DeInit(&DCMI_MDMA_Handle0);
         HAL_MDMA_DeInit(&DCMI_MDMA_Handle1);
         #endif
+        // Reset all relevant DCMI flags.
+        __HAL_DCMI_CLEAR_FLAG(&DCMIHandle, DCMI_FLAG_FRAMERI);
+        __HAL_DCMI_CLEAR_FLAG(&DCMIHandle, DCMI_FLAG_VSYNCRI);
+        __HAL_DCMI_DISABLE_IT(&DCMIHandle, DCMI_IT_FRAME);
```

Comment:
You should disable the interrupt first, then clear the flags if you want to be thorough because it can interrupt after the flag is cleared...see the IRQ handler in the HAL driver.

Reply:
The interrupt is already cleared here. However, it may be disabled.